src/test/java/org/opensrp/service/ProductCatalogueServiceTest.java

```diff
@@ -40,8 +40,9 @@ public void testFindAllProductCatalogs() {
 		ProductCatalogueSearchBean productCatalogueSearchBean = new ProductCatalogueSearchBean();
 		productCatalogueSearchBean.setProductName("");
 		productCatalogueSearchBean.setUniqueId(0l);
+		int noLimit = Integer.MAX_VALUE;
 		List<ProductCatalogue> productCatalogueList = productCatalogueService
-				.getProductCatalogues(productCatalogueSearchBean, "http://localhost:8080/opensrp");
+				.getProductCatalogues(productCatalogueSearchBean, noLimit, "http://localhost:8080/opensrp");
```

Comment:
I think we should add new test cases for this and still leave the unit test for the deprecated methods in place (until they are removed)